rviz_visual_tests/docs/README.md

```diff
@@ -0,0 +1,89 @@
+# A quick start guide: how to write and run RViz visual tests
+A deeper explanation on what this testing framework is about and on how it works can be found in
+[this document](documentation.md).
+
+The following serves as a quick introduction to the framework.
+
+## Writing a test
+
+We will look at the following simple example:
+
+```cpp {.line-numbers}
+TEST_F(VisualTestFixture, example_test_structure) {
+      /// Set the position of the camera and its sight vector (optional):
+      setCamPose(Ogre::Vector3(0, 3, 16));
+      setCamLookAt(Ogre::Vector3(0, 2, 0));
+
+      /// Add displays:
+      auto grid_display = addDisplay<GridDisplayPageObject>();
+      auto image_display = addDisplay<ImageDisplayPageObject>();
+
+      /// Modify their properties:
+      grid_display->setOffset(0.3, 2, 0.4);
+      grid_display->setColor(0, 255, 0);
+      grid_display->collapse();
+      image_display->setQueueSize("10");
+
+      /// Take the screenshots of the desired render windows:
+      captureMainWindow();
+      captureRenderWindow(image_display);
+
+      /// Compare test screenshots with the reference ones:
+      assertScreenShotsIdentity();
+    }
```

Comment:
nitpick: the indentation of this code snippet looks weird, either the first line should be indented or the rest should be unindented once.

Reply:
The example test looks really nice, btw :D